packages/web/src/components/tooltip/Tooltip.tsx

```diff
@@ -23,16 +23,38 @@ export const Tooltip = ({
   shouldWrapContent = true,
   text = ''
 }: TooltipProps) => {
-  const [hideTooltip, setHideTooltip] = useState(false)
+  // Keep track of a hidden state ourselves so we can dismiss the tooltip on click
+  const [isHidden, setIsHidden] = useState(false)
 
-  useEffect(() => {
-    if (hideTooltip) {
-      const hideTooltipTimeout = setTimeout(() => {
-        setHideTooltip(false)
-      }, 2500)
-      return () => clearTimeout(hideTooltipTimeout)
+  // Keep track of whether we are hovering over the tooltip to know when to
+  // allow it to become visible again
+  const mousedOver = useRef(false)
+
+  const onClick = useCallback(() => {
+    if (shouldDismissOnClick) {
+      setIsHidden(true)
     }
-  }, [hideTooltip])
+  }, [shouldDismissOnClick, setIsHidden])
+
+  const onMouseOut = useCallback(() => {
+    mousedOver.current = false
+    setIsHidden(false)
```

Comment:
should this be hiding the tooltip or un-hiding it?

Reply:
this is setting the overriding hidden state to false, which is different from actually hiding it. I can rename the state var to make it more understandable